pkg/operator/handlers/dapr_handler_test.go

```diff
@@ -0,0 +1,111 @@
+package handlers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	testclient "github.com/dapr/dapr/pkg/client/clientset/versioned"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestGetDaprID_WithValidId(t *testing.T) {

```

Comment:
Can you group tests per method with `t.Run()`?

See this reference:
https://github.com/dapr/dapr/blob/4c350c552066970be62cd1c15e155ffe38350e79/pkg/http/api_test.go#L58